tools/authoring-workflows.adoc

```diff
@@ -1,37 +1,37 @@
 = Authoring Workflows
 include::../partials/attributes.adoc[]
 
-Workflows authoring on {CANVAS} happens inside the Editor page for Workflow, where you can edit, and validate BPMN files.
+Workflow authoring on {CANVAS} happens inside the Editor page for Workflow, where you can edit and validate BPMN files.
 
-Let's start by describing the elements present on the BPMN Editor.
+Let's start by describing the elements present in the BPMN Editor.
 
 .Editor page for a Workflow on {CANVAS} with highlighted elements of the BPMN Editor
 image::tools/image5.jpg[]
 
-. *Node palette* → From here you can select node types and its variations.
+. *Node palette* → From here you can select node types and their variations.
 . *Side panel strip* → Lets you select which side panel you want to toggle.
   From top to bottom, those are the available panels:
 [loweralpha]
 .. Properties → Lets you manage the properties of a Workflow element.
 .. Navigator → Displays a minimap with the diagram of your Workflow and a list of all elements present on it.
-. *Keyboard shortcuts panel toggle →* Clicking here toggles a panel with information of keyboard shortcuts available on the BPMN Editor.
+. *Keyboard shortcuts panel toggle →* Clicking here toggles a panel with information about keyboard shortcuts available in the BPMN Editor.
 
 Toggling the Problems tab lets you see static validation errors that {CANVAS} reports while you're developing your Workflows.
 
-Developing Workflows through BPMN files is a complex subject and has many intricacies that are out of the scope of this document.
+Developing Workflows through BPMN files is a complex subject and has many intricacies that are beyond the scope of this document.
 Please refer to BPMN-specific material, like the https://www.omg.org/spec/BPMN/2.0.2/PDF[BPMN specification] itself.
 
 == Invoking a Decision from a Workflow
 
-A common use-case for Workflows is to invoke Decisions as part of its logic.
+A common use case for a Workflow is to invoke Decisions as part of its logic.
 Luckily, it's very easy to achieve that on {CANVAS}.
 On your Workflow, create a new “Business Rule” task node.
 
 .Node palette open for Task nodes inside the BPMN Editor on {CANVAS}
 image::tools/image9.png[]
 
 Open the “Properties” panel using the first icon at the “Side panel strip” and look for Implementation/Execution.
-There, you can change the “Rule language” to “DMN”. Select the File you want and the Decision inside it.
+There, you can change the “Rule language” to “DMN”. Select the file you want and the Decision inside it.
```

Comment:
```suggestion
There, you can change the "Rule language" to "DMN". Select the file you want and the Decision inside it.
```

just to be consistent with other adoc files

Reply:
Good old quote issues :)